command/agent.go

```diff
@@ -321,9 +321,15 @@ func (c *AgentCommand) Run(args []string) int {
 			}
 		}
 
+		// Check if a default namespace has been set
+		mountPath := config.AutoAuth.Method.MountPath
+		if config.AutoAuth.Method.Namespace != "" {
+			mountPath = fmt.Sprintf("%s%s", config.AutoAuth.Method.Namespace, mountPath)
```

Comment:
Might be cleaner to use `path.Join` here to remove any potential malformed paths (e.g. missing slash, double slashes, etc).

Reply:
TIL `path.Join` exists. Thanks!